ui/server/src/main/scala/pl/touk/nussknacker/ui/config/package.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.ui
+
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import net.ceedubs.ficus.readers.ValueReader
+
+package object config {
+
+  implicit object Implicits extends LazyLogging {
```

Comment:
Maybe let's add comment that it's only to provide debug info, otherwise we'd use getAs[T]?